src/components/v5/common/ActionSidebar/partials/TokenSelect/hooks.tsx

```diff
@@ -7,13 +7,16 @@ import { SpinnerLoader } from '~shared/Preloaders';
 import TokenIcon from '~shared/TokenIcon/TokenIcon';
 import { formatText } from '~utils/intl';
 import { SearchSelectOptionProps } from '~v5/shared/SearchSelect/types';
-import TokenStatus from './partials/TokenStatus/TokenStatus';
 import { useGetAllTokens } from '~hooks/useGetAllTokens';
 import { useColonyContext } from '~hooks';
+import { useAdditionalFormOptionsContext } from '~context/AdditionalFormOptionsContext/AdditionalFormOptionsContext';
```

Comment:
Why does this context even exist, it's just a single value. Surely there's a better way to handle that.

As a second issue, the naming of both the hook and the actual prop name are problematic in the sense that you cannot infer specificity from them

Reply:
@rdig this context is updated in [this PR](https://github.com/JoinColony/colonyCDapp/pull/1483/files#diff-aa1d9524e92861e0fbf79d94d32aaa3f1b29b8cc124019c66c88024c864ae82f) 
If you have a suggestion on how it should be done in a different way, please share.